azure/setup-env-variables-template.sh

```diff
@@ -1,15 +1,14 @@
-export SUBSCRIPTION=subscription-id                 # replace it with your subscription-id
-export RESOURCE_GROUP=resource-group-name           # existing resource group or one that will be created in next steps
-export SPRING_APPS_SERVICE=azure-spring-apps-name   # name of the service that will be created in the next steps
-export LOG_ANALYTICS_WORKSPACE=log-analytics-name   # existing workspace or one that will be created in next steps
-export REGION=region-name                           # choose a region with Enterprise tier support
-
-export CART_SERVICE_APP="cart-service"
-export IDENTITY_SERVICE_APP="identity-service"
-export ORDER_SERVICE_APP="order-service"
-export PAYMENT_SERVICE_APP="payment-service"
-export CATALOG_SERVICE_APP="catalog-service"
-export FRONTEND_APP="frontend"
-
-export CUSTOM_BUILDER="no-bindings-builder"
-
+   export SUBSCRIPTION=4274ebdf-fab7-4711-8332-afab2fef9aeb                 # replace it with your subscription-id
+   export RESOURCE_GROUP=kelwin-rg           # existing resource group or one that will be created in next steps
+   export SPRING_APPS_SERVICE=kelwin-asa   # name of the service that will be created in the next steps
+   export LOG_ANALYTICS_WORKSPACE=log-analytics-kelwin   # existing workspace or one that will be created in next steps
```

Comment:
let's make these names generic, looks great other than this.

Reply:
done!